homeassistant/components/homely/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""The homely integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .homely_device import HomelyHome
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.BINARY_SENSOR, Platform.SENSOR]
```

Comment:
Usually we only accept a single platform for new integration merges
https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=single&_highlight=platform#5-make-your-pull-request-as-small-as-possible

Reply:
I am not sure what to do about this. Is it okay as it is, or should I remove one of the platforms and put in a second pull request afterwards?

Reply:
Correct, only a single platform in the initial PR is allowed

Reply:
I have removed the binary sensor platform.